modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectLayoutLocalServiceImpl.java

```diff
@@ -570,10 +574,32 @@ private void _validate(
 						"Object layout box must have a type");
 				}
 
+				if (StringUtil.equals(
+						objectLayoutBox.getType(),
```

Comment:
Hi @marcelabc, I believe this condition is not necessary because we only have two ObjectLayoutBox types and after check if the type is categorization, the loop will continue in the same way. If we do this, we are adding one more condition to check when the ObjectLayoutBox type is categorization. 

What do you think about this suggestion?